acceptance/features/release.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Golden Container Image
+
+    Scenario: Red Hat collection
+        Given there is a sample golden-container policy input
+        And there is a policy config
+            """
+            {
+                "sources": [
+                    {
+                        "policy": [
+                            "$GITROOT/policy/lib",
+                            "$GITROOT/policy/release"
+                        ],
+                        "data": [
+                            "$GITROOT/example/data"
+                        ],
+                        "config": {
+                            "include": [
+                                "@redhat"
+                            ],
+                            "exclude": [
+                                "sbom_cyclonedx.found",
+                                "redhat_manifests.redhat_manifests_missing",
+                                "cve.deprecated_cve_result_name"
+                            ]
+                        }
+                    }
+                ]
+            }
+            """
+        When input is validated with policy config
```

Comment:
```suggestion
        When input is validated
```

Or like this, and then ditch the `And there is a policy config` line above:

```suggestion
        When input is validated with policy config:
```

Reply:
https://github.com/enterprise-contract/ec-policies/pull/866